### PR TITLE
feat: add support for font awesome on theme creation

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: packages/generator-liferay-theme/generators/app/index.js [hunk 1]
```diff
@@ -196,6 +196,11 @@ module.exports = class extends Generator {
 					instance._isLiferayVersion
 				),
 			},
+			{
+				message: 'Would you like to add font awesome to your theme?',
```
Comment (wincent) [inline packages/generator-liferay-theme/generators/app/index.js]: Let's make that "Font Awesome".

File: packages/liferay-theme-tasks/tasks/build.js [hunk 2]
```diff
@@ -256,6 +257,27 @@ module.exports = function(options) {
 			.pipe(gulp.dest(pathBuild + '/css'));
 	});
 
+	gulp.task('build:copy:fontAwesome', done => {
+		const themePath = process.cwd();
+
+		const packageJSON = require(path.join(themePath, 'package.json'))
+			.liferayTheme;
+
+		if (packageJSON.fontAwesome !== 'true') {
```
Comment (wincent) [inline packages/liferay-theme-tasks/tasks/build.js]: Let's make this a boolean.

**Comment by mwilliams2014** [comment]
@wincent what version will this feature be available in, and do you know when?

**Comment by mwilliams2014** [comment]
Ah, 9.3.0, nevermind, :-)

**Pull request merged**